services/brig/src/Brig/Whitelist.hs

```diff
@@ -1,44 +1,68 @@
 -- | > docs/reference/user/activation.md {#RefActivationWhitelist}
 --
 -- Email/phone whitelist.
-module Brig.Whitelist (Whitelist (..), verify) where
+module Brig.Whitelist
+    ( Whitelist (..)
+    , InternalWhitelist (..)
+    , verifyInternal
+    , verifyService
+    ) where
 
 import Imports
 import Data.Aeson
+import Data.Aeson.TH (deriveJSON)
 import Bilge.IO
 import Bilge.Request
 import Bilge.Response
 import Bilge.Retry
 import Brig.Types
 import Control.Monad.Catch (MonadMask, throwM)
 import Control.Retry
-import Data.Text
+import qualified Data.Text as T
```

Comment:
Import lists are generally (across the codebase) split into two lists, one with `import ...` and one with `import qualified ... as ...`, separated by a blank line. See for example [here](https://github.com/wireapp/wire-server/blob/develop/services/brig/src/Brig/API.hs#L33-L40)

Reply:
Done.